server/saved_objects/observability_saved_object.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { SavedObjectsType } from '../../../../src/core/server';
+import { observabilityID } from '../../common/constants/shared';
+import { VISUALIZATION_SAVED_OBJECT } from '../../common/types/observability_saved_object_attributes';
+
+export const visualizationSavedObject: SavedObjectsType = {
+  name: VISUALIZATION_SAVED_OBJECT,
+  hidden: false,
+  namespaceType: 'single',
+  management: {
+    defaultSearchField: 'title',
+    importableAndExportable: true,
+    icon: 'visQueryPPL',
+    getTitle(obj) {
+      return obj.attributes.title;
+    },
+    getInAppUrl(obj) {
+      const editPath = `#/event_analytics/explorer/${VISUALIZATION_SAVED_OBJECT}:${obj.id}`;
+      const editUrl = `/app/${observabilityID}${editPath}`;
+      return {
+        path: editUrl,
+        uiCapabilitiesPath: 'observability.show',
+      };
+    },
+  },
+  mappings: {
+    dynamic: false,
+    properties: {
+      title: {
+        type: 'text',
+      },
+      description: {
+        type: 'text',
+      },
+      version: { type: 'integer' },
+    },
+  },
```

Comment:
Are we finalizing on mappings or is that subject to change?

Reply:
this can change, will panels have mappings for specific fields?